bciers/apps/compliance/src/app/components/compliance-summary/manage-obligation/apply-compliance-units/ApplyComplianceUnitsComponent.tsx

```diff
@@ -170,6 +170,11 @@ export default function ApplyComplianceUnitsComponent({
     if (!response || response.error) {
       setStatus("idle");
       setErrors([response?.error || "Failed to get compliance units data"]);
+    } else if (response.bccr_trading_name === null) {
+      setStatus("idle");
+      setErrors([
+        "Please enter a valid BCCR Holding Account ID that owns the compliance sub-account for this operation.",
```

Comment:
We’re already raising this error:
`raise UserError("The provided holding account does not own the compliance sub-account for this operation.")`
We could reuse it instead of having two separate error messages serving the same purpose.